packages/celotool/src/lib/attestation.ts

```diff
@@ -187,7 +194,13 @@ export async function chooseFromAvailablePhoneNumbers(
   maximumNumberOfAttestations: number,
   salt: string,
 ) {
-  const availableNumbers = await twilioClient.incomingPhoneNumbers.list()
+  const availableNumbers = (await twilioClient.incomingPhoneNumbers.list()).filter(
+    (number) => number.smsUrl === DUMMY_SMS_URL
+  )
+  if (!availableNumbers?.length) {
+    return undefined
+  }
```

Comment:
Wouldn't findSuitableNumber handle the case where the availableNumbers is an empty list due to filtering?

Reply:
I saw cases where it wasn't an empty list, but `[null]`